python/lsst/ts/watcher/alarm.py

```diff
@@ -223,6 +230,42 @@ def close(self):
         self._cancel_escalation_timer()
         self._cancel_unmute()
 
+    async def make_log_entry(self, log_server_url):
+        """Post message to narrative log entry in response to alarm"""
+        now = datetime.datetime.now().isoformat()  # use ts_utils repo?
```

Comment:
Instead of using `date time` , use `utils.current_tai()`.

You can get it iso-formatted with:

```
now = utils.astropy_time_from_tai_unix(utils.current_tai()).isoformat()
```

Reply:
Changed.  But is there a reason for this requested change?  The original does it more simply.  Using the standard python library makes it easier for someone unfamiliar to all the ts code to know what it does.

Reply:
I tried your alternative.  But it got an error:
> AttributeError: 'Time' object has no attribute 'isoformat'

I reverted back to use datetime.

Reply:
Sorry, I missed one thing on my suggested code, it should be:

```
now = utils.astropy_time_from_tai_unix(utils.current_tai()).datetime.isoformat()
```

There are a couple reasons I think this is a better alternative;

1. In general we use the lsst.ts utility to manage timestamps.
2. It will make sure the timestamp has the correct format (TAI).
3. You don't need to import any additional library.